dev/helm-chart-starter/values.yaml

```diff
@@ -0,0 +1,168 @@
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+# Set up pod health monitoring probes, see https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+# livenessProbe:
+#  httpGet:
+#    path: v1/health_check
+#    port: <CHARTNAME>
+#  initialDelaySeconds: 5
+#  periodSeconds: 5
+#  failureThreshold: 24
+readinessProbe:
+  httpGet:
+    path: v1/health_check
+    port: <CHARTNAME>
+  initialDelaySeconds: 5
+  periodSeconds: 5
+startupProbe:
+  httpGet:
+    path: v1/health_check
+    port: <CHARTNAME>
+  initialDelaySeconds: 5
+  periodSeconds: 5
+  failureThreshold: 120
+
+# This section is for setting up autoscaling more information can be found here: https://kubernetes.io/docs/concepts/workloads/autoscaling/
+# Note: Do not use (above) "replicaCount" with HPA (Chart ignores value=1 as it's k8s default)
+# Note: Because HPA can sometimes change replica counts up and down rather frequently, the microservice needs to handle SIGTERM elegantly:
+# - stop accepting new requests
+# - handle all of its buffered requests
+# - terminate after tthose have been processed
+# See https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination.
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 4
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+# Additional volumes on the output Deployment definition.
+# volumes:
+#  - name: tmp
+#    emptyDir: {}
+#  - name: foo
+#    secret:
+#      secretName: mysecret
+#      optional: false
+
+# Additional volumeMounts on the output Deployment definition.
+# volumeMounts:
+#  - name: tmp
+#    mountPath: /tmp
+#  - name: foo
+#    mountPath: "/etc/foo"
+#    readOnly: true
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+global:
+  http_proxy: ""
+  https_proxy: ""
+  no_proxy: ""
+  HUGGINGFACEHUB_API_TOKEN: "insert-your-huggingface-token-here"
+  # service account name to be shared with all parent/child charts.
+  # If set, it will overwrite serviceAccount.name.
+  # If set, and serviceAccount.create is false, it will assume this service account is already created by others.
+  sharedSAName: ""
+  # Install Prometheus serviceMonitor for service metrics
```

Comment:
`# Assumes service to have `/metrics` endpoint providing Prometheus metrics.  If service uses OPEA HttpService class, it does`